internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -95,7 +95,12 @@ func convertSidecars(s map[string]*manifest.SidecarConfig, exposedPorts map[stri
 	sort.Strings(keys)
 	for _, name := range keys {
 		config := s[name]
-		imageURI, _ := config.ImageURI()
+		var imageURI string
+		if uri, buildRequired := config.ImageURI(); buildRequired {
+			imageURI = uri
```

Comment:
Wut I'm so confused what's happening here 😅 
reading the doc comments for `ImageURI` it says:
```
// ImageURI returns the location of the image if one is set.
// If the image needs to be build, return "" and false.
```

So should this be something like (the opposite of a "build being required"):
```suggestion
		if uri, hasLocation := config.ImageURI(); hasLocation {
			imageURI = uri
```

Reply:
sorry for the confusion 😅. yes,this should be renamed.